### PR TITLE
add progress

### PR DESCRIPTION
**Pull request by TheGreatRandall**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -2409,7 +2409,7 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-com
...
 https://github.com/apache/nifi,41ff6f076b2e5346cf695ff2064e5dcfe249c124,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.wali.EncryptedSequentialAccessWriteAheadLogTest.testShouldUpdateWithExternalFile,OD,,,
 https://github.com/apache/nifi,41ff6f076b2e5346cf695ff2064e5dcfe249c124,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.wali.EncryptedSequentialAccessWriteAheadLogTest.testShouldUpdateWithExternalFileAfterCheckpoint,OD,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services,org.apache.nifi.xml.TestXMLReader.testAttributePrefix,ID,Accepted,https://github.com/apache/nifi/pull/6709,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services,org.apache.nifi.xml.TestXMLReader.testAttributePrefix,ID,Opened,https://github.com/apache/nifi/pull/6676,
```
Comment (darko-marinov) [inline pr-data.csv:2412]: I don't understand this. It seems you're opening a new PR for a test for which another PR was already accepted.

File: pr-data.csv [hunk 2]
```diff
@@ -2990,7 +2990,7 @@ https://github.com/espertechinc/esper,590fa9c9eb854f1420b9d337b802aca19f963cc0,r
 https://github.com/fabiomaffioletti/jsondoc,de5bd0024b4e5444a541fa4fa4ec518176b646f3,jsondoc-core,org.jsondoc.core.scanner.builder.JSONDocApiAuthBuilderTest.testApiAuthToken,ID,Accepted,https://github.com/fabiomaffioletti/jsondoc/pull/261,
 https://github.com/facebook/facebook-java-business-sdk,492cda9aa1bac6e04e1b80de1794e74f9cea1cb6,facebook-java-business-sdk,com.facebook.ads.sdk.serverside.UserDataTest.BuilderAndGettersWithMultipleValuesTest,ID,Opened,https://github.com/facebook/facebook-java-business-sdk/pull/359,
 https://github.com/facebook/facebook-java-business-sdk,492cda9aa1bac6e04e1b80de1794e74f9cea1cb6,facebook-java-business-sdk,com.facebook.ads.ServerSideTest.CustomDataParametersTest,ID,Opened,https://github.com/facebook/facebook-java-business-sdk/pull/360,
-https://github.com/facebook/facebook-java-business-sdk,4d9fd05f8d3909d23093fbd65c6b2b84c32f1ba9,.,com.facebook.ads.ServerSideTest.CustomDataParametersTest,ID,,,
+https://github.com/facebook/facebook-java-business-sdk,4d9fd05f8d3909d23093fbd65c6b2b84c32f1ba9,.,com.facebook.ads.ServerSideTest.CustomDataParametersTest,ID,failWithMvnCompile,,
```
Comment (darko-marinov) [inline pr-data.csv:2993]: There's no status `failWithMvnCompile`, but you can add to notes which Java version you tried and which project version you tried.

File: pr-data.csv (hunk 1 above)
Comment (TheGreatRandall) [reply pr-data.csv:2412]: Hi Porfessor. I signed this test on excel first and I did not see anyone else signed for this. Thank you!

Comment (xieyt2000) [reply pr-data.csv:2412]: > Hi Porfessor. I signed this test on excel first and I did not see anyone else signed for this. Thank you!

Is this test still flaky in the latest commit `2fa82179a81877c975333ba1315f9431330780f6`?

**Comment by TheGreatRandall** [comment]
[error.log](https://github.com/TestingResearchIllinois/idoft/files/10202712/error.log)
Here are 4 mvn install -pl eror logs. Thank you

**Comment by TheGreatRandall** [comment]
the newest pr only have the opened and accepted flaky test. Thank you!

**Comment by xieyt2000** [comment]
> the newest pr only have the opened and accepted flaky test. Thank you!

I still see some `Skipped` changes, such as `https://github.com/spring-projects/spring-data-couchbase,6c6acf8ae2940a6e479c8cbe3413094d04a55aa1,.,org.springframework.data.couchbase.core.mapping.MappingCouchbaseConverterTests.idHasAnnotatedIdAndMultipleIdFields,ID,Skipped,,`

Also, please revert this change `https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services,org.apache.nifi.xml.TestXMLReader.testAttributePrefix,ID,Opened,https://github.com/apache/nifi/pull/6676,
Contributor`

**Comment by TheGreatRandall** [comment]
> > the newest pr only have the opened and accepted flaky test. Thank you!
> 
> I still see some `Skipped` changes, such as `https://github.com/spring-projects/spring-data-couchbase,6c6acf8ae2940a6e479c8cbe3413094d04a55aa1,.,org.springframework.data.couchbase.core.mapping.MappingCouchbaseConverterTests.idHasAnnotatedIdAndMultipleIdFields,ID,Skipped,,`
> 
> Also, please revert this change `https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services,org.apache.nifi.xml.TestXMLReader.testAttributePrefix,ID,Opened,https://github.com/apache/nifi/pull/6676, Contributor`

Hi Yuntong,
You should see the correct file without skipped right now.

**Review by xieyt2000 — APPROVED** [review]

**Comment by darko-marinov** [comment]
Accepting per Yuntong's approval.

**Pull request merged**